src/PortToTripleSlash/tests/PortToTripleSlash/PortToTripleSlash.Strings.Tests.cs

```diff
@@ -0,0 +1,1423 @@
+    /// <remarks>These are the MyGenericMethod remarks.</remarks>
+    public T MyGenericMethod<T>(T withGenericArgument);
+    /// <summary>This is the MySetProperty summary.</summary>
+    /// <value>This is the MySetProperty value.</value>
+    /// <remarks>These are the MySetProperty remarks.</remarks>
+    public double MySetProperty { set; }
+    /// <summary>This is the MyGetProperty summary.</summary>
+    /// <value>This is the MyGetProperty value.</value>
+    /// <remarks>These are the MyGetProperty remarks.</remarks>
+    public double MyGetProperty { get; }
+    /// <summary>This is the MyGetSetProperty summary.</summary>
+    /// <value>This is the MyGetSetProperty value.</value>
+    /// <remarks>These are the MyGetSetProperty remarks.</remarks>
+    public double MyGetSetProperty { get; set; }
+}";
+
+        List<string> docFiles = new() { docFile };
+        List<string> originalCodeFiles = new() { originalCode };
+        Dictionary<string, string> expectedCodeFiles = new() { { docId, expectedCode } };
+        StringTestData stringTestData = new(docFiles, originalCodeFiles, expectedCodeFiles, false);
+
+        return TestWithStringsAsync(stringTestData);
+    }
+
+    private static Task TestWithStringsAsync(StringTestData stringTestData) =>
+        TestWithStringsAsync(new Configuration() { SkipInterfaceImplementations = false }, DefaultAssembly, stringTestData);
+
+    private static async Task TestWithStringsAsync(Configuration c, string assembly, StringTestData data)
+    {
+        Assert.True(data.XDocs.Any(), "No XDoc elements passed.");
+        Assert.True(data.OriginalCodeFiles.Any(), "No original code files passed.");
+        Assert.True(data.ExpectedCodeFiles.Any(), "No expected code files passed.");
+
+        c.IncludedAssemblies.Add(assembly);
+
+        CancellationTokenSource cts = new();
+
+        CSharpCompilationOptions compileOptions = new(outputKind: OutputKind.DynamicallyLinkedLibrary, optimizationLevel: OptimizationLevel.Release, allowUnsafe: true);
+
+        List<SyntaxTree> syntaxTrees = new();
+
+        CSharpParseOptions parseOptions = new(languageVersion: LanguageVersion.Preview, kind: SourceCodeKind.Regular);
+        foreach (string originalCode in data.OriginalCodeFiles)
+        {
+            CompilationUnitSyntax pcu = SyntaxFactory.ParseCompilationUnit(originalCode, options: parseOptions);
+            syntaxTrees.Add(pcu.SyntaxTree);
+        }
+
+        CSharpCompilation compilation = CSharpCompilation.Create(assembly, options: compileOptions)
+            .AddSyntaxTrees(syntaxTrees);
+
+        // Use only when it is expected to inherit documentation from inherited implementations of .NET APIs
+        if (data.AddMsCorLibReferences)
+        {
+            // reference same mscorlib we're running on
+            compilation = compilation.AddReferences(MetadataReference.CreateFromFile(typeof(object).Assembly.Location));
```

Comment:
This gave me so many headaches. Turns out the `Add*` methods under the `CSharpCompilation` return a new instance and needs to override the old instance. Otherwise, your change is not reflected.